### DISCUSSION TITLE
With BRX releasing, is it possible for future Original Xbox support?

### DISCUSSION THREAD

**Discussion by colton-everett** [darthcloud/BlueRetro#1086] [Q&A] [answered]
I haven't seen anyone explicitly ask here, so I just wanted to confirm. With the release of BRX using BlueRetro, does that mean that support for Original Xbox could be possible down the line? I am building an AIO unit and would love to also have it set up for my Xbox without having to buy 4 BRXs or building a OG360.

**Comment by darthcloud** [comment, marked as answer]
While BRX is base on BlueRetro software wise the hardware itself is completely different and totally incompatible with BlueRetro.

So no, Xbox support will not come to BlueRetro.

**Reply by colton-everett** [reply]
Bummer. Thanks for confirming!
